packages/iTwinUI-react/src/core/Header/HeaderButton.tsx

```diff
@@ -113,6 +115,7 @@ export const HeaderButton: HeaderButtonComponent = React.forwardRef(
       <li
         className='iui-header-breadcrumb-item'
         aria-current={isActive ? 'location' : undefined}
+        aria-disabled={disabled ? 'true' : undefined}
```

Comment:
Can you add disabled button to at least one of our stories? Would be good to have test for it :)

Reply:
And verify that all others work (basic, split and dropdown)

Reply:
I disabled the split button in the Full Header story. These are what each of the following disabled buttons look like (split, dropdown, and basic)

![image](https://user-images.githubusercontent.com/78564221/185956113-70ae2e97-0916-46a9-864b-57832593e101.png)